src/VoteDelegateFactory.sol

```diff
@@ -59,7 +59,7 @@ contract VoteDelegateFactory {
     }
 
     function create() external returns (address voteDelegate) {
-        voteDelegate = address(new VoteDelegate{salt: keccak256(abi.encode(msg.sender))}(chief, polling, msg.sender));
```

Comment:
Hmm shouldn't we just avoid the `msg.sender` as also proposed before?

Reply:
Their final recommendation was to do it this way.
While both would probably work I think it makes sense for the msg.sender to be the salt, acting as a unique identifier for the vote-delegate.
Gas is less of an issue here (vote-delegate creations are pretty rare, and the diff should be minor anyway).

Reply:
Hmmm, not really sure why the value of the salt should really matter, but if feeling more comfortable leaving it then fine. I was a bit surprised they left only that option as recommendation.